resources.whatwg.org/build/deploy.sh

```diff
@@ -111,6 +112,32 @@ run_post_build_step "$WEB_ROOT"
 echo "Living standard output to $WEB_ROOT"
 echo ""
 
+header "Starting review drafts..."
+echo "Note: review drafts must be added or changed in a single commit on master"
+for f in "$REVIEW_DRAFTS_DIR"/*.bs; do
+    [ -e "$f" ] || continue # http://mywiki.wooledge.org/BashPitfalls#line-80
+    if [[ "$TRAVIS_PULL_REQUEST" == "true" ]]; then
+        CHANGED_FILES=$(git diff --name-only master..HEAD)
+    else
+        CHANGED_FILES=$(git diff --name-only HEAD~1)
+    fi
+    for change in "$CHANGED_FILES"; do
+        if [[ f != change ]]; then
+            continue
+        fi
+        echo ""
+        BASENAME=$(basename "$f" .bs)
+        DRAFT_DIR="$WEB_ROOT/$REVIEW_DRAFTS_DIR/$BASENAME"
+        mkdir -p "$DRAFT_DIR"
+        curlbikeshed -F md-Status="RD" \
```

Comment:
Should this be whatwg/RD?

Reply:
Nah, since we're already in the WHATWG org, just the status suffices.